specs/spec-2-election-ui.md

```diff
@@ -0,0 +1,200 @@
+- Results -> green check mark
+- Votes can not be changed once cast, but can be changed before they are. 
+
+## Tech Spec
+
+<!-- How does the product implement the features and requirements? Not every feature or requirement needs deep consideration here, but implementation considerations are discusssed they should be captured here. -->
+
+### Smart Contract Functions
+
+<!-- What functions and functionalities should the widget have -->
+
+### Data Structures
+
+### UML Diagrams
+
+<!-- Include diagrams that best capture what needs to be built. Sequence, State, Interaction, Activity, Etc.-->
+
+### Technical Dependencies 
+
+Voting Contracts @ https://github.com/near-ndc/voting-v1
+
+## Design Specification
+
+<!-- UX and UI that affords all of the features and requirements. How it behaves and what it looks like. -->
+
+### Wireframes / Screens
+
+- [2023-05-30 v0.1.0](https://www.figma.com/file/OTKmz4ga22khc2vtWPFkzB/Election-UI-Wireframe?type=design&node-id=102%3A2&t=rNzg02hFvCuYLv9a-1)
+  - Desktop Voting for review
+  - Roughed out Desktop Profile & Desktop Admin
+
+### UI Design
+
+Follow the NDC Brand Book in line with Material Design, falling back on vanilla [Bootstrap](https://getbootstrap.com/).
+
+![Primary and Secondary NDC Colors in a Material Design Pallet](https://i.imgur.com/Rzx9XnX.jpg)
+
+### Dependent Widgets
+
+<!-- Does the widget interact with other widgets? -->
+
+- Kudos Widget
+- Near Social Profile
+
+## Open Questions and Comments
+
+<!-- Topics not covered in this document that need to be addressed in a meeting or async. Potential future improvements, Summary of open discussion. -->
+### Open
+
+### Resoloved
+
+- **Q:** How long is the election voting window? If no comment: 8 days (long enough that someone can take a week vacation and still participate in governance). 
+  - **A:** Election timing will be retrieved from the smart contract. It will be set on the smart contract by NDC administrators.  
+- **Q:**Considering votes are on-chain and don’t lack privacy, should we be able to see votes? In the spec above we are assuming yes. Going off of this, we are unclear on whether a voting body member can override their vote for a congressional body before the deadline as it would make the vote even more susceptible to swindling.
+  - **A:** Yes, we will be displaying votes
+- **Q:** One of the biggest challenges for NDC Voting body members is to notify them of a vote. THis brings an overall question of outside of BOS Push notification, and what point of the I-AM-HUMAN onboarding flow will users optin and enable Web2 notifications (in a GDPR compliant fashion) so we can remind them of voting (and other NDC critical updates) outside of BOS. This may result in a separate spec for overall NDC GDPR Compliant Notifications
+  - **A:** We will use standard BOS notifications.
+- **Q:** What is our runoff policy? 
+  - **A:** An NDC administrator will begin a new runoff election in the case of a tie. 
```

Comment:
We don't have NDC administrator. How need to define the role, how about Voting Comission, Voting Comissioner, or GWG (need to check in GWG about more opinions)

Reply:
Starting elections is handled entirely on the contract side via CLI. The voting administrators will be a list of addresses on the voting contract who have permissions to call these functions.